guide/06-imagery-and-raster-analysis/lidar_basics_part1.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p style="user-select: auto;">Change to "Working with LiDAR data" -&gt; Heading level 1</p><p style="user-select: auto;">"Part 1 - Introductory concepts" -&gt; Heading level 2</p><p style="user-select: auto;"><br></p><p style="user-select: auto;">Let us will keep the level1 heading same for all notebooks in this series and change the level 2 for each notebook. Let us align the filename of the notebook with the level 2 heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>